x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -310,6 +310,25 @@ const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
           },
         ],
       },
+      '3': {
+        changes: [
+          {
+            type: 'mappings_addition',
+            addedMappings: {
+              secrets: {
+                properties: {
+                  service_token: {
+                    dynamic: false,
+                    properties: {
+                      id: { type: 'keyword' },
+                    },
+                  },
+                },
+              },
```

Comment:
@elastic/fleet fixing the mapping that was incorrectly added without a model version in https://github.com/elastic/kibana/pull/171875

Reply:
@pgayvallet I wanted to confirm that the cli check detects problems and ran `node scripts/check_mappings_update.js --task mapping-addition` locally after removing `modelVersion 3` from here.
The check didn't respond with any issues. I'm assuming it should though. Is there some extra flag we need that's not documented well or is something else going on.

FYI: I did bootstrap after removing the model version.